presto-product-tests/src/main/java/io/prestosql/tests/hive/TestExternalHiveTable.java

```diff
@@ -151,6 +151,18 @@ public void testCreateExternalTableWithInaccessibleSchemaLocation()
         query(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation));
     }
 
+    @Test(groups = {HIVE_WITH_EXTERNAL_WRITES, PROFILE_SPECIFIC_TESTS})
+    public void testCreateExternalTableWithNoSchemaLocation()
+    {
+        String schema = "schema_without_location";
+        query(format("DROP SCHEMA IF EXISTS %s.%s", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema));
+        query(format("CREATE SCHEMA %s.%s", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema));
```

Comment:
Won't this schema have an implicit location of `/usr/hive/warehose/schema_without_location.db` ?

Reply:
Yeah, I want to create a test case for AWS Glue for create an external table, where creating a schema has no location clause, nor an implicit location.

Reply:
We have Glue-specific tests in TestHiveGlueMetastore.

Reply:
Ok, thanks for the tip. I will take a look and update the tests accordingly.